timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -618,8 +626,27 @@ export class TimeGraphChart extends TimeGraphChartLayer {
             if (el instanceof TimeGraphStateComponent && !this.mousePanning && !this.mouseZooming) {
                 this.selectState(el.model);
             }
-            if (this.mouseInteractions && this.mouseInteractions.click) {
+            
+            var self = this;
+            this._mouseClicks++;
+
+            if (this._mouseClicks === 1) {
+                this._recentlyClickedGlobal = cloneDeep(e.data.global);
```

Comment:
E.g. 1) Time selection through mouse dragging calls the following events:
mouseover -> mousedown -> mouseup -> click

 E.g. 2) Time selection using shift + left mouse click calls the following events:
mouseover -> mousedown -> mouseup -> click -> mousedown -> mouseup -> click

Click events are always sent at the end of time selection, but they would not trigger doubleClick since 'e.data.global' would capture different positions as cursor positions have changed during drag. But I feel that in  E.g. 1 (mouse dragging) we should prevent click event being triggered